frontend/package.json

```diff
@@ -153,7 +153,7 @@
     "nodemon": "^2.0.16",
     "raw-loader": "0.5.1",
     "ssgrtk": "^0.3.4",
-    "testcafe": "^2.6.0",
+    "testcafe": "^3.1.0",
```

Comment:
Is this meant to be in this PR?

Reply:
I can split this if we want to review it separately, but really if the tests pass I think it can go through

Reply:
I would rather we keep things like that separate if possible so we have atomic commits, but I'm not _too_ fussy here.